packages/web-app-files/src/mixins/actions/emptyTrashBin.js

```diff
@@ -27,12 +32,35 @@ export default {
     }
   },
   methods: {
-    ...mapActions(['showMessage']),
+    ...mapActions(['showMessage', 'createModal', 'hideModal', 'toggleModalConfirmButton']),
     ...mapActions('Files', ['clearTrashBin']),
+
     $_emptyTrashBin_trigger() {
-      this.$client.fileTrash
-        .clearTrashBin()
+      const modal = {
+        variation: 'danger',
+        title: this.$gettext('Empty trash bin'),
+        cancelText: this.$gettext('Cancel'),
+        confirmText: this.$gettext('Delete'),
+        icon: 'alarm-warning',
+        message: this.$gettext(
+          'Are you sure you want to permanently delete your items in the trash bin? You can’t undo this action.'
```

Comment:
```suggestion
          'Are you sure you want to permanently delete the items in the trash bin of this space? You can’t undo this action.'
```

Reply:
Thx, this will require a decision between personal and spaces trash bin as we are using the same action here. 
but no problem.